scripts/welcome.js

```diff
@@ -200,7 +200,11 @@ var link_repo = (token, name)=>{
                     /* Change mode type to commit */
                     chrome.storage.sync.set({"mode_type": "commit"}, data=>{
                         $("#error").hide();
-                        $("#success").html(`Successfully linked <a target="blank" href="${res['html_url']}">${name}</a> to LeetHub. Start <a href="http://leetcode.com">LeetCoding</a> now!`);
+                        $("#success").html(`Successfully linked <a target="blank" href="${res['html_url']}">${name}</a> to LeetHub. Start <a href="http://leetcode.com">LeetCoding</a> now! <br /> Linked the wrong repo? <a id="unlink">Unlink</a>.`);
+                        $("#unlink").on('click', () => {
```

Comment:
see comment below, declare globally on document load. text will be hidden if invalid status code is present anyways. thoughts?